### DISCUSSION TITLE
21.1: Your wish has been granted 🤣

### DISCUSSION THREAD

**Discussion by Secret-chest** [SebastJava/mint-themes#48] [General]
![colors](https://user-images.githubusercontent.com/74449186/205969891-329872da-7089-4a32-84f9-d8021a47e339.png)
I don't use Mint-Y but it's nice

**Comment by fblais** [comment]
Yep, the new colors in Mint 21.1 are wonderful!
I guess those are Sébastien's work!  :)

**Comment by SebastJava** [comment]
> Your wish has been granted :)

Yes, it seems someone finally heard me when i was saying those old colors made me feel bad and unmotivated. Now that's much better.

> I guess those are Sébastien's work! :)

This is not 100% "my colors" but yes, i clearly see i had a positive influence in there. I like those new colors. They are much better.

---

I just have a little doubt about the reaction coming from people who prefer to live in the shade, as expressed in https://github.com/linuxmint/mint-themes/issues/414

I don't know, i am just expressing some doubts... I personally usually prefer the light themes and these are just undoubtedly much better now!
